helpers.go

```diff
@@ -70,3 +73,85 @@ func getListOfVersionedRepositories(inVersion string) ([]string, error) {
 
 	return strings.Split(strings.TrimSpace(string(output)), "\n"), nil
 }
+
+func validDependabotPR(pr *github.PullRequestEvent) bool {
+	return pr.GetAction() == "opened" && pr.GetSender().GetLogin() == "dependabot[bot]"
+}
+
+func maybeVendorDependabotPR(pr *github.PullRequestEvent) error {
+	if !validDependabotPR(pr) {
+		log.Debugf("Not a valid dependabot PR (%s:%s), ignoring...\n", pr.GetSender().GetLogin(), pr.GetAction())
+		return nil
+	}
+
+	branchName := pr.GetPullRequest().GetHead().GetRef()
+	if branchName == "" {
+		return fmt.Errorf("No Head set in the PR: %v", pr)
+	}
+
+	repo := pr.GetRepo().GetName()
+	if repo == "" {
+		return fmt.Errorf("No repository name found in PR: %v", pr)
+	}
+
+	tmpdir, err := ioutil.TempDir("", repo)
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(tmpdir)
+
+	cmd := exec.Command("git", "clone", "--single-branch", "--branch", branchName, "git@github.com:mendersoftware/"+repo+".git")
+	cmd.Dir = tmpdir
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("%v returned error: %s: %s", cmd.Args, out, err.Error())
+	}
+
+	cmdDir := tmpdir + "/mender-artifact"
```

Comment:
`mender-artifact`? I don't understand...

Reply:
Ah, I see, you need `repo` here instead.

Reply:
lol, fixing

Reply:
Good thing we have more eyes on this

Reply:
thx